src/gf.c

```diff
@@ -874,14 +878,15 @@ JL_DLLEXPORT int jl_isa_compileable_sig(
         }
```

Comment:
I vague think the previous conditional (`jl_is_kind(elt)`) should `return 0` if `iscalled`, but there's a TODO there that it's possibly missing other cases already